alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -0,0 +1,89 @@
+# PrometheusOperatorRejectedResources
+
+## Meaning
+
+The `PrometheusOperatorRejectedResources` alert triggers when Prometheus Operator
+rejects  `Alertmanager`, `PodMonitor`, `ServiceMonitor`, or `PrometheusRules` objects
+because custom resources in an object are not configured correctly.
+
+## Impact
+
+The custom resources that trigger the alert are ignored by Prometheus Operator.
+As a consequence, they will not be part of the final configuration of the
+Prometheus, Alertmanager, or Thanos Ruler components managed by Prometheus Operator.
+
+## Diagnosis
+
+Find the type of custom resource and its namespace in the `description` field
+of the `PrometheusOperatorRejectedResources` alert. You can find this information
+by using the command line interface (CLI) or the OpenShift web console.
+
+### Diagnose using the CLI
+
+```bash
+### Gets alertmanager URL
+$ ALERT_MANAGER=$(oc get route alertmanager-main -n openshift-monitoring -o jsonpath='{@.spec.host}')
+
+### Gets fired alerts
+$ curl -k -H "Authorization: Bearer $(oc create token prometheus-k8s -n openshift-monitoring)"  https://$ALERT_MANAGER/api/v1/alerts
+```
+
+### Diagnose using the Openshift web console
+
+1. Browse to **Observe** -> **Alerting**.
+2. Click the **Filter** drop-down list and select **Firing**.
+3. Search for the `PrometheusOperatorRejectedResources` alert.
+4. Click the alert to view its details.
+5. Scroll down to see the **Description** field. The namespace is displayed as
+a link, followed by the resource type.
+
+After you know the namespace and the resource type, you can look for a recently
+modified custom resource of that type in that namespace.
+**Note**: If the namespace is named `openshift-.*`, (excluding`openshift-user-workload-monitoring`),
```

Comment:
the namespace value is either openshift-monitoring (in which case they would need to contact the support as you've written) or openshift-user-workload-monitoring for user-defined resources.